src/mmcache.hpp

```diff
@@ -230,7 +237,25 @@ if (cache[hidx].finger_print_cnt_sum <= 5)
         cache[hidx].strands.resize(0);
         return;
       }
-
+      int size = pos_candidates.size();
+      int shift = (int)minimizers[0].second >> 1;
+      // Do not cache if it is too near the start.
+      for (i = 0; i < size; ++i) 
+        if ((int)pos_candidates[i].position < kmer_length + shift) {
+          cache[hidx].offsets.resize(0);
```

Comment:
Can we use clear() instead of resize(0) here?

Reply:
I think resize is in general faster than clear, and we do want to keep the allocated memory in cache for potential future updates.

Reply:
But if you want to keep the memory, you have to use clear() instead of resize(), which actually release the memory?